app/src/main/res/values/strings.xml

```diff
@@ -96,7 +96,7 @@
     <!-- Browser menu toggle that installs a Progressive Web App shortcut to the site on the device home screen. -->
     <string name="browser_menu_install_on_homescreen">Install</string>
     <!-- Menu option on the toolbar that takes you to synced tabs page-->
-    <string name="synced_tabs">Synced Tabs</string>
+    <string name="synced_tabs_2">Synced tabs</string>
     <!-- Browser menu button that opens the find in page menu -->
```

Comment:
Please see https://github.com/mozilla-mobile/fenix/commit/44d9dc9237e054349da832487a3d94384a2663e8#r39455402

> changing capitalization to existing strings doesn't require to use new IDs. The reason is that, by changing the ID, we require all languages to retranslate. But each language has its own capitalization rules, so they should already be using them consistently, independently from what English does.

Reply:
@cadeyrn thanks for the reminder! 
@flodolo if we don't change the ID, so we don't retranslate the strings, there will be languages (for example English CA) that will still keep the wrong capitalisation. So is there a way to say: hey, there are multiple languages that don't respect the correct capitalisation rule, please look over them, without changing the ID?

Reply:
There's no way, but you can ping me and I can fix the English variants directly.

Reply:
got it, thanks!